.github/workflows/build.yml

```
@@ -2,23 +2,23 @@ name: build-test
-    branches: [ main ] 
+    branches: [main, develop]
   pull_request:
-    branches: [ main ]
+    branches: [main, develop]
 jobs:
   build-and-test:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)